src/Widgets/DataFormUploadWidget/DataFormUploadWidgetComponent.tsx

```diff
@@ -112,12 +137,18 @@ provideComponent(DataFormUploadWidget, ({ widget }) => {
         />
         {getDropMessage(widget.get('multiple'))}
       </div>
-      {isTooLarge && (
+      {rejectionErrors.tooLarge && (
         <div>
           <i className="bi bi-exclamation-diamond" aria-hidden="true" />{' '}
           {getTooLargeMessage()}
         </div>
       )}
+      {rejectionErrors.tooMany && (
+        <div>
+          <i className="bi bi-exclamation-diamond" aria-hidden="true" />{' '}
```

Comment:
* [ ] FYI The icon looks weird. It would look better with just the text. But:
* [ ] FYI Since this is a temporary error message, maybe a toast would work better?

<img width="587" height="128" alt="Image" src="https://github.com/user-attachments/assets/ff844575-2d5c-4cc4-9274-d89e099e134d" />